session/src/main/java/org/apache/iotdb/session/Session.java

```diff
@@ -2442,6 +2461,24 @@ private TSDropSchemaTemplateReq getTSDropSchemaTemplateReq(String templateName)
     }
   }
 
+  /** Transmit insert record request for double write */
+  public void doubleWriteTransmit(ByteBuffer buffer)
```

Comment:
what's this for?

Reply:
To transmit all types of the insert plan, and to avoid re-transmit.